dhcp/install.sh

```diff
@@ -2,12 +2,33 @@
 
 source /vagrant/settings.sh
 
-#installation
+# installation
 apt --yes update
 apt --yes --install-recommends install dnsmasq
+export DEBIAN_FRONTEND=noninteractive
+apt -yq --yes install iptables-persistent
 
-#configuration
+# fetching interface name
+INTERNET=$(ifconfig | grep -m 1 RUNNING | cut -d':' -f1)
+LOCAL=$(ifconfig | grep -m 2 RUNNING | cut -d':' -f1 | tail -n1)
```

Comment:
Here try something like this:
`ip route | grep -v default | cut -d' ' -f3 | grep -v $INTERNET | head -1`